pkg/apis/extensions/types.go

```diff
@@ -482,6 +468,20 @@ type DaemonSetStatus struct {
 	// ObservedGeneration is the most recent generation observed by the daemon set controller.
 	// +optional
 	ObservedGeneration int64
+
+	// UpdatedNumberScheduled is the total number of nodes that are running updated
+	// daemon pod
+	UpdatedNumberScheduled int32
+
+	// NumberAvailable is the number of nodes that should be running the
+	// daemon pod and have one or more of the daemon pod running and
+	// available (ready for at least minReadySeconds)
+	NumberAvailable int32
+
+	// NumberUnavailable is the number of nodes that should be running the
```

Comment:
FWIW, for Deployments, this is the number of pods running but not considered ready/available.
https://github.com/kubernetes/kubernetes/blob/d953402cdf414a877cb537d8db46646db38e7bef/pkg/controller/deployment/sync.go#L586